exporter/exporter_test.go

```diff
@@ -26,6 +26,18 @@ import (
 	"github.com/percona/mongodb_exporter/internal/tu"
 )
 
+// Use this for testing because labels like cluster ID are not constant in docker containers
```

Comment:
why don't we use some mock library for this?

Reply:
Because this is super simple. If I add an mock library, I would need to create a mock and functions everywhere it is being used or a function to return the mocked struct. There is no real gain on doing that.